src/components/ProductNav.astro

```diff
@@ -39,7 +42,15 @@ const getStartedHref = nav.entries.filter(
 		<div class="flex items-center gap-4 ml-auto flex-none text-stone-200">
 			<a href={DISCORD_URL} class="flex items-center gap-1.5 hover:text-white">
 				<Icon name="fa-brands:discord" class="w-5 h-5" />
-				<span class="sr-only md:not-sr-only">Community</span>
+				<span class="sr-only md:not-sr-only"></span>
+			</a>
+			<a
+				href={product.github}
+				class="rounded-md flex items-center bg-stone-700 p-1 gap-1.5 hover:text-white"
```

Comment:
I'm not fully sold on the background. Perhaps a translucent border would be better?

<img width="287" alt="image" src="https://github.com/leaningtech/labs.leaningtech.com/assets/9429556/debd6ffd-7b39-429b-8dc2-f461d90754a3">

Reply:
I have pushed a different style